ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/package.scala

```diff
@@ -0,0 +1,109 @@
+package com.daml.ledger.participant.state
+
+import com.daml.ledger.participant.state.kvutils.DamlKvutils._
+import com.daml.ledger.participant.state.v1.{
+  Configuration,
+  SubmittedTransaction,
+  SubmitterInfo,
+  TransactionMeta,
+  Update
+}
+import com.digitalasset.daml.lf.data.Time.Timestamp
+import com.digitalasset.daml.lf.engine.Engine
+import com.digitalasset.daml_lf.DamlLf.Archive
+
+/** The participant-state key-value utilities provide methods to succintly implement
+  * participant-state on top of ledger's that provide a key-value state storage.
```

Comment:
the participant-state Read/WriteService interfaces

Reply:
provide motivation for why we somebody should use this tooling. Also explain why we provide this tooling (to enable evolution of DAML).